package.json

```diff
@@ -1,30 +1,52 @@
 {
   "name": "fast-bmp",
+  "type": "module",
```

Comment:
Was this part of the TS migration docs? The published package won't work with it.